packages/amplify_auth_cognito/ios/Classes/Utils/UserAttributeDeserialization.swift

```diff
@@ -50,6 +50,12 @@ func createAuthUserAttributeKey(keyName: String) -> AuthUserAttributeKey {
     case "preferred_username":
         return AuthUserAttributeKey.preferredUsername
     default:
-        return AuthUserAttributeKey.custom(keyName)
+        // amplify-ios does not currently include enums for these keyNames
```

Comment:
Note: This isn't needed for the new functionality, but it addresses a related bug.